static/app/views/dashboardsV2/widgetCard/issueWidgetQueries.tsx

```diff
@@ -92,28 +111,101 @@ class WidgetQueries extends React.Component<Props, State> {
     }
   }
 
-  transformTableResults(tableResults: Group[]): TableDataRow[] {
-    return tableResults.map(({id, shortId, title, assignedTo, ...resultProps}) => {
+  assignableTeams(group: Group): AssignableTeam[] {
+    if (!group) {
+      return [];
+    }
+
+    const teams = ProjectsStore.getBySlug(group.project.slug)?.teams ?? [];
+    return teams
+      .sort((a, b) => a.slug.localeCompare(b.slug))
+      .map(team => ({
+        id: buildTeamId(team.id),
+        display: `#${team.slug}`,
+        email: team.id,
+        team,
+      }));
+  }
+
+  getSuggestedAssignees(group: Group): SuggestedAssignee[] {
+    const {memberList} = this.props;
+    const {owners: suggestedOwners} = group;
+    if (!suggestedOwners || !memberList) {
+      return [];
+    }
+
+    const assignableTeams = this.assignableTeams(group);
+    const suggestedAssignees: Array<SuggestedAssignee | null> = suggestedOwners.map(
```

Comment:
Does none of this code already exist somewhere else? Didn't @getsentry/workflow team build out a whole suggested assignee thing as well?

Reply:
👍  spoke with @evanpurkhiser over slack and agreed to update and reuse `AssigneeSelector` instead. This duplicate code has been removed!